### ISSUE TITLE
Crash during YouTube Live

### ISSUE THREAD

**Issue by shogo4405** [shogo4405/lf.swift#47]
```sh
[fg211,211,211;[bg;2016-05-24 23:58:01.261 [Verbose] [RTMPSocket.swift:38] doOutput(chunk:) > RTMPChunk{size:0,type:One,streamId:4,message:Optional(RTMPAudioMessage{type:Audio,length:0,streamId:1,timestamp:21,payload:[175, 1, 1, 24, 20, 169, 208, 118, 74, 12, 16, 64, 68, 8, 54, 67, 11, 169, 112, 93, 124, 23, 48, 123, 182, 247, 44, 29, 246, 153, 242, 65, 91, 145, 164, 86, 83, 75, 197, 76, 167, 117, 137, 175, 73, 135, 82, 29, 70, 219, 168, 90, 236, 176, 215, 93, 103, 119, 4, 141, 110, 76, 80, 138, 203, 151, 54, 206, 57, 138, 145, 188, 205, 220, 87, 103, 211, 17, 51, 11, 81, 3, 20, 151, 132, 248, 192, 1, 96, 10, 111, 30, 199, 207, 225, 182, 88, 194, 87, 108, 45, 163, 89, 228, 93, 38, 164, 13, 121, 237, 224, 130, 42, 110, 241, 68, 157, 69, 90, 82, 195, 41, 56, 149, 133, 155, 188, 48, 6, 35, 64, 212, 11, 70, 197, 179, 11, 105, 171, 59, 10, 173, 35, 22, 178, 25, 230, 88, 165, 196, 98, 163, 172, 239, 240, 245, 140, 213, 32, 164, 19, 80, 40, 95, 94, 231, 66, 84, 107, 99, 27, 93, 248],config:nil,codec:Unknown,soundR2016-05-24 23:58:01.378 Example MacOS[5398:633050] Dropping audio sample buffer 0x102817590 for output 
[fg255,0,0;[bg;2016-05-24 23:58:01.322 [Error] [RTMPMessage.swift:59] create > 198[;
```
```sh
[fg255,0,0;[bg;2016-02016-05-26 06:48:21.562 Example MacOS[11017:1396736] Dropping audio sample buffer 0x100b9c350 for output <AVCaptureAudioDataOutput: 0x60800003edc0> connection <AVCaptureConnection: 0x608000008e90 [type:soun][enabled:1][active:1]>
5-26 06:48:21.468 [Error] [RTMPMessage.swift:59] create > 23[;
[fg255,0,0;[bg;2016-05-26 06:48:21.563 [Error] [RTMPChunk.swift:213] bytes > [130, 0, 0, 0, 0, 4, 2, 23][;
```

**Comment by Victor1506** [comment]
I am also have a crash:
2016-05-25 11:08:23.803 [Error] [RTMPMessage.swift:61] create > 104
2016-05-25 11:08:23.806 [Error] [RTMPChunk.swift:128] bytes > [130, 0, 0, 0, 0, 4, 9, 104]
warning: could not load any Objective-C class information. This will significantly reduce the quality of type information available.

**Comment by shogo4405** [comment]
@Victor1506 
fixed it. https://github.com/shogo4405/lf.swift/commit/8a7b3ff38d496fb8df5151657d833ddfcc211a10

**Comment by shogo4405** [comment]
released 0.3.1

**Comment by shogo4405** [comment]
released 0.3.2
